bin/curriculum/delete_unit.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'json'
 require 'optparse'
+require 'ostruct'
```

Comment:
remove unused require?

Reply:
sounds good. sometimes when I copy/paste my IDE adds imports like this that I may or may not need.